{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.23.1",
      "sacred==0.8.2",
      "torch==1.12.1"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [
      {
        "commit": "302fb40048c482b990c7763ce820c08b85d161bd",
        "dirty": true,
        "url": "https://github.com/JonasWild/epymarl.git"
      },
      {
        "commit": "302fb40048c482b990c7763ce820c08b85d161bd",
        "dirty": true,
        "url": "https://github.com/JonasWild/epymarl.git"
      },
      {
        "commit": "302fb40048c482b990c7763ce820c08b85d161bd",
        "dirty": true,
        "url": "https://github.com/JonasWild/epymarl.git"
      }
    ],
    "sources": [
      [
        "main.py",
        "_sources\\main_6484b33f523008f82bae719361a1abc7.py"
      ],
      [
        "run.py",
        "_sources\\run_ea3c26a018523bc70547637375beee5f.py"
      ],
      [
        "utils\\logging.py",
        "_sources\\logging_7ddc698bb74674bae897bfdc5662ed00.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"C:\\Users\\jonas\\.conda\\envs\\pymarl\\lib\\site-packages\\sacred\\config\\captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\src\\main.py\", line 49, in my_main\n    run(_run, config, _log)\n",
    "  File \"C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\src\\run.py\", line 60, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\src\\run.py\", line 225, in run_sequential\n    runner.run(test_mode=True)\n",
    "  File \"C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\src\\runners\\episode_runner.py\", line 73, in run\n    reward, terminated, env_info = self.env.step(actions[0])\n",
    "  File \"C:\\Users\\jonas\\Documents\\epymarl\\epymarl\\smacv2\\smacv2\\env\\starcraft2\\starcraft2.py\", line 635, in step\n    self._obs = self._controller.observe()\n",
    "  File \"C:\\Users\\jonas\\.conda\\envs\\pymarl\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 95, in _valid_status\n    return func(self, *args, **kwargs)\n",
    "  File \"C:\\Users\\jonas\\.conda\\envs\\pymarl\\lib\\site-packages\\pysc2\\lib\\stopwatch.py\", line 205, in _stopwatch\n    return func(*args, **kwargs)\n",
    "  File \"C:\\Users\\jonas\\.conda\\envs\\pymarl\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 252, in observe\n    raise ValueError(\"Expect a player result in a stub observation\")\n",
    "ValueError: Expect a player result in a stub observation\n"
  ],
  "heartbeat": "2023-06-20T12:35:38.432633",
  "host": {
    "ENV": {},
    "cpu": "Unknown",
    "gpus": {
      "driver_version": "531.79",
      "gpus": [
        {
          "model": "NVIDIA GeForce GTX 960",
          "persistence_mode": false,
          "total_memory": 2048
        }
      ]
    },
    "hostname": "DESKTOP-1HDPUO6",
    "os": [
      "Windows",
      "Windows-10-10.0.22621-SP0"
    ],
    "python_version": "3.8.16"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2023-06-20T08:12:55.424220",
  "status": "FAILED",
  "stop_time": "2023-06-20T12:35:38.554216"
}